[2025-03-30T00:26:10.747+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-30T00:26:10.914+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T00:26:08.287940+00:00 [queued]>
[2025-03-30T00:26:10.923+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T00:26:08.287940+00:00 [queued]>
[2025-03-30T00:26:10.924+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-30T00:26:10.938+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-03-30 00:26:08.287940+00:00
[2025-03-30T00:26:10.947+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1070) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-30T00:26:10.949+0000] {standard_task_runner.py:72} INFO - Started process 1089 to run task
[2025-03-30T00:26:10.949+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_data_pipeline', 'fetch_data', 'manual__2025-03-30T00:26:08.287940+00:00', '--job-id', '296', '--raw', '--subdir', 'DAGS_FOLDER/Gestioncommands.py', '--cfg-path', '/tmp/tmppyn673up']
[2025-03-30T00:26:10.950+0000] {standard_task_runner.py:105} INFO - Job 296: Subtask fetch_data
[2025-03-30T00:26:10.997+0000] {task_command.py:467} INFO - Running <TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T00:26:08.287940+00:00 [running]> on host 499696d11f67
[2025-03-30T00:26:11.082+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='ecommerce_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T00:26:08.287940+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T00:26:08.287940+00:00'
[2025-03-30T00:26:11.084+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-30T00:26:11.085+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-30T00:26:11.086+0000] {logging_mixin.py:190} INFO - Current task name:fetch_data state:running start_date:2025-03-30 00:26:10.915870+00:00
[2025-03-30T00:26:11.087+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline and current dag run status:running
[2025-03-30T00:26:11.088+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-30T00:26:11.089+0000] {Gestioncommands.py:40} INFO - Fetching data from API...
[2025-03-30T00:26:11.709+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host 'my.api.mockaroo.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-03-30T00:26:12.484+0000] {Gestioncommands.py:45} INFO - Data fetched successfully
[2025-03-30T00:26:12.486+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-30T00:26:12.495+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-30T00:26:12.495+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ecommerce_data_pipeline, task_id=fetch_data, run_id=manual__2025-03-30T00:26:08.287940+00:00, execution_date=20250330T002608, start_date=20250330T002610, end_date=20250330T002612
[2025-03-30T00:26:12.516+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-30T00:26:12.517+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-30T00:26:12.517+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline queued_at:2025-03-30 00:26:08.304953+00:00
[2025-03-30T00:26:12.517+0000] {logging_mixin.py:190} INFO - Task hostname:499696d11f67 operator:PythonOperator
[2025-03-30T00:26:12.533+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-30T00:26:12.558+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T00:26:12.561+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
